From a33d684078557ba8231145c76f19c587e51c4565 Mon Sep 17 00:00:00 2001
From: Hauke Mehrtens <hauke.mehrtens@intel.com>
Date: Wed, 8 Mar 2017 10:34:31 +0100
Subject: selftests: ptp: fix build with musl libc

musl libc does not define __GLIBC_PREREQ, but it supports clock_adjtime().
Add it only if __GLIBC_PREREQ is supported.

Signed-off-by: Hauke Mehrtens <hauke.mehrtens@intel.com>
---
 tools/testing/selftests/ptp/testptp.c | 2 ++
 1 file changed, 2 insertions(+)

--- a/tools/testing/selftests/ptp/testptp.c
+++ b/tools/testing/selftests/ptp/testptp.c
@@ -49,6 +49,7 @@
 #endif
 
 /* clock_adjtime is not available in GLIBC < 2.14 */
+#ifdef __GLIBC_PREREQ
 #if !__GLIBC_PREREQ(2, 14)
 #include <sys/syscall.h>
 static int clock_adjtime(clockid_t id, struct timex *tx)
@@ -56,6 +57,7 @@ static int clock_adjtime(clockid_t id, s
 	return syscall(__NR_clock_adjtime, id, tx);
 }
 #endif
+#endif
 
 static clockid_t get_clockid(int fd)
 {
